src/chrome/content/rules/Univision.com.xml

```diff
@@ -0,0 +1,483 @@
+			- miembros-hotfix.uim.univision.com
+			- noticias-test.uim.univision.com
+			- noticias-ti11.uim.univision.com
+			- novelasyseries-ti8.uim.univision.com
+			- proxy2.uim.univision.com
+			- s-hotfix.uim.univision.com
+			- h.s-test.uim.univision.com
+			- videos-hotfix.uim.univision.com
+			- h.www-test.uim.univision.com
+			- www2-qa4.uim.univision.com
+			- www2-test.uim.univision.com
+			- www2-uat.uim.univision.com
+			- uimapi.univision.com
+			- umc.univision.com
+			- mx171.umc.univision.com
+			- m.univisionchicago.univision.com
+			- m.univisiondallas.univision.com
+			- activesync.us.univision.com
+			- connectra.us.univision.com
+			- host120.us.univision.com
+			- lasmtp60.us.univision.com
+			- miami194.us.univision.com
+			- miami210.us.univision.com
+			- miami211.us.univision.com
+			- miami212.us.univision.com
+			- miami221.us.univision.com
+			- miasmtp.us.univision.com
+			- ny130.us.univision.com
+			- ny154.us.univision.com
+			- ny58.us.univision.com
+			- remote.us.univision.com
+			- submail2.us.univision.com
+			- wes.univision.com
+			- wojo.univision.com
+			- wviv.univision.com
+			- ww.univision.com
+			- h.www.univision.com
+			- www2.univision.com
+			- www3.univision.com
+			- xauth.univision.com
+			- youtoo.univision.com
+			- www.uvnimg.com
+			- s0.uvnimg.com
+			- www.s0.uvnimg.com
+			- s0-hotfix.uvnimg.com
+			- s1-hotfix.uvnimg.com
+			- h.s1-test.uvnimg.com
+		Unexpected http status code:
+			- auth.univision.com
+			- origin.auth.univision.com
+			- feedsyn.univision.com
+			- kiswe.univision.com
+			- lakallechicago.univision.com
+			- luisjimenez.univision.com
+			- musicgroup.univision.com
+			- nomvauth.univision.com
+			- cdn.performance.univision.com
+			- cdn0.performance.univision.com
+			- cdn1.performance.univision.com
+			- cdn2.performance.univision.com
+			- cdn3.performance.univision.com
+			- cdn4.performance.univision.com
+			- cdn5.performance.univision.com
+			- cdn7.performance.univision.com
+			- cdn8.performance.univision.com
+			- cdn9.performance.univision.com
+			- cms.performance.univision.com
+			- syndicator.performance.univision.com
+			- origin.syndicator.performance.univision.com
+			- recuerdodallas.univision.com
+			- static.univision.com
+			- syndicator.univision.com
+			- api.vmh.dev.tvsa.univision.com
+			- uforiamusica.univision.com
+			- usaauth.univision.com
+			- usaplusauth.univision.com
+			- usprauth.univision.com
+			- api.vmh.univision.com
+			- webmail.univision.com
+			- auth.dev.y.univision.com
+			- nomvauth.dev.y.univision.com
+			- prauth.dev.y.univision.com
+			- usaauth.dev.y.univision.com
+			- usaplusauth.dev.y.univision.com
+			- usprauth.dev.y.univision.com
+			- usprcauth.dev.y.univision.com
+			- api.vmh.dev.y.univision.com
+			- cdn1.uvnimg.com
+			- cdn2.uvnimg.com
+			- cdn3.uvnimg.com
+			- cdn4.uvnimg.com
+			- lmx1.uvnimg.com
+-->
+<ruleset name="Univision.com">
+	<target host="www.univision.com" />
```

Comment:
Please add a comment for `uvnimg.com` and a target for `univision.com` (without `www`).